src/analysis/Environments/Environments.ts

```diff
@@ -280,10 +280,14 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
       includeLabels: 'saturnApplication,saturnWorkspaceNamespace,saturnWorkspaceName',
     };
 
+    const cromwellAppTypes = ['CROMWELL', 'WORKFLOWS_APP', 'CROMWELL_RUNNER_APP'];
     const [newRuntimes, newDisks, newApps] = await Promise.all([
       leoRuntimeData.list(listArgs, { signal }),
       leoDiskData.list(diskArgs, { signal }),
-      leoAppData.listWithoutProject(listArgs, { signal }),
+      // Excluding cromwell/workflows app types for now, as trying to delete these app types puts associated workspace in an error state
+      (
+        await leoAppData.listWithoutProject(listArgs, { signal })
```

Comment:
Hmm... I don't particularly like this approach, it is going the opposite direction we are going, adding business logic to visual components.

I'd advise waiting until [this PR](https://github.com/DataBiosphere/terra-ui/pull/4585/files#diff-2249b30f7fc3880b0b3dd2ea8b365aa35f632e4b1bba40f37db91e0e6cf30719) is merged, and extending `environmentPermissions` with the already existing function `isResourceDeletable`. 

cc @msilva-broad , so you know I'm advising someone extend work you have in progress